lib/animation.dart

```diff
@@ -34,6 +36,9 @@ class Animation {
   /// Whether the animation loops after the last sprite of the list, going back to the first, or keeps returning the last when done.
   bool loop = true;
 
+  /// Registered method to be triggered when the animation finish's, this variable is set by the method addOnCompleteAnimation
+  OnCompleteAnimation _onCompleteAnimation;
```

Comment:
We don't usually make fields private as you can see on the loop before, I think I would prefer to keep that pattern.

@luanpotter and @renancaraujo your thoughts?

Reply:
Ok I usually use private fields when the follow variable is not useful fot the outside world, like in this case that its just used to trigger the registered method.

But please if its not the pattern that you guys use in this project, let me know so I can refactor to be as you guys use to do it.